tendrl/node_agent/objects/definition/master.py

```diff
@@ -285,6 +285,126 @@
+    Message:
+      attrs:
+        message_id:
+          help: "Message UUID"
+          type: String
+        timestamp:
+          help: "time"
+          type: Time
+        priority:
+          help: "Message priority"
+          type: String
+        publisher:
+          help: "Message publisher"
+          type: String
+        node_id:
+          help: "node id"
+          type: String
+        payload:
+          help: "Differ based on message"
+          type: Dict
+        caller:
+          help: "Called details"
+          type: Dict
+      enabled: true
+      list: /Messages
+      help: "Messages"
+      value: /Messages
+    NodeMessage:
+      attrs:
+        message_id:
+          help: "Message UUID"
+          type: String
+        timestamp:
+          help: "time"
+          type: Time
+        priority:
+          help: "Message priority"
+          type: String
+        publisher:
+          help: "Message publisher"
+          type: String
+        node_id:
+          help: "node id"
+          type: String
+        payload:
+          help: "Differ based on message"
+          type: Dict
+        caller:
+          help: "Called details"
+          type: Dict
+      enabled: true
+      list: node/$Node_context.node_id/Messages
+      help: "Node Messages"
+      value: node/$Node_context.node_id/Messages
+    ClusterMessage:
+      attrs:
+        message_id:
+          help: "Message UUID"
+          type: String
+        timestamp:
+          help: "time"
+          type: Time
+        priority:
+          help: "Message priority"
+          type: String
+        publisher:
+          help: "Message publisher"
+          type: String
+        node_id:
+          help: "node id"
+          type: String
+        payload:
+          help: "Differ based on message"
+          type: Dict
+        caller:
+          help: "Called details"
+          type: Dict
+      enabled: true
+      list: clusters/$TendrlContext.integration_id/Messages
+      help: "Cluster Messages"
+      value: clusters/$TendrlContext.integration_id/Messages
+    JobUpdate:
+      attrs:
+        message_id:
+          help: "Message UUID"
+          type: String
+        timestamp:
+          help: "time"
+          type: Time
+        priority:
+          help: "Message priority"
+          type: String
+        publisher:
+          help: "Message publisher"
+          type: String
+        node_id:
+          help: "node id"
+          type: String
+        payload:
+          help: "Differ based on message"
+          type: Dict
+        request_id:
+          help: "Job id"
+          type: String
+        flow_id:
+          help: "Flow id"
+          type: String
+        parent_id:
+          help: "parent id"
+          type: String
+        cluster_id:
+          help: "cluster id"
+          type: String
+        caller:
+          help: "Called details"
+          type: Dict
+      enabled: true
+      list: $request_id
```

Comment:
@shtripat job updates stored only using request id, then this should br the key right?